resources/views/layout/welcome.blade.php

```diff
@@ -0,0 +1,108 @@
+
+<!DOCTYPE html>
+
+<html lang="en">
+
+<head>
+
+
+ <link rel="stylesheet" type="text/css" href="css/index.css">
+
+
+ <!-- jQuery library -->
+ <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+ <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"/>
+ <style>
```

Comment:
Remove this code here, and copy it to `resources/sass/footer.scss`.
Then in `app.scss` add ```@import 'footer';```